lib/Alien/Build/MM.pm

```diff
@@ -422,7 +422,11 @@ sub mm_install
     $mm->SUPER::install(@rest);
   };
 
-  "install :: alien_clean_install\n\n$section";
+  return
+      ".NOTPARALLEL : install\n\n"
```

Comment:
```suggestion
      ".NOTPARALLEL :\n\n"
```
https://www.gnu.org/software/make/manual/html_node/Special-Targets.html
Any prerequisites on this target are ignored.

Reply:
Done.